### ISSUE TITLE
iterable: New value type (array) is not in annotated types.

### ISSUE THREAD

**Issue by heisch** [kalessil/phpinspectionsea#844]
There's a false positive with `iterable` as parameter type and arrays

This is the [valid](https://wiki.php.net/rfc/iterable) (7.1+) php script:
```php
function foo(?iterable $foo) {
    $foo = $foo ?? [];
}
```

and this is the warning from PHPInspectionsEA:
![screen shot 2018-03-08 at 09 56 21](https://user-images.githubusercontent.com/3083118/37142024-26894222-22b7-11e8-8ced-07a68a50215e.png)

**Comment by kalessil** [comment]
Looks like a bug, I'll fix it in the next release.

**Comment by heisch** [comment]
Thanks a lot and keep up the good work 👍

**Comment by kalessil** [comment]
Thank you for reporting and you are welcome =)

**Comment by kalessil** [comment]
Fixed
